website/script.js

```diff
@@ -1,33 +1,34 @@
+function createDate(year, month, day, time = null) {
+    // Creates a moment object with the specified date and time (optional) in America/Toronto timezone
+    const dateStr = `${year}-${month}-${day}` + (time ? ` ${time}` : '');
+    return moment.tz(dateStr, time ? 'YYYY-MM-DD HH:mm' : 'YYYY-MM-DD', 'America/Toronto');
+}
+
+function createToday(){
+    return moment.tz('America/Toronto')
+}
```

Comment:
nit: we could take in the timezone as an argument as well which would default to America/Toronto

Reply:
Hmm ... I  don't think we'll ever need it, and I think that `createDate` and `createToday` are modular enough that doing it in the future would be pretty easy.

Reply:
I appreciate your detail oriented comment